radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/transport/PeerChannel.java

```diff
@@ -154,35 +154,36 @@ public PeerChannel(
       Optional<RadixNodeUri> uri,
       SocketChannel nettyChannel,
       Optional<InetSocketAddress> remoteAddress) {
-    this.counters = Objects.requireNonNull(counters);
-    this.addressing = Objects.requireNonNull(addressing);
-    this.peerEventDispatcher = Objects.requireNonNull(peerEventDispatcher);
-    this.uri = Objects.requireNonNull(uri);
-    uri.ifPresent(u -> this.remoteNodeId = u.getNodeId());
+    this.counters = requireNonNull(counters);
+    this.addressing = requireNonNull(addressing);
+    this.peerEventDispatcher = requireNonNull(peerEventDispatcher);
+    this.uri = requireNonNull(uri);
+
+    uri.map(RadixNodeUri::getNodeId).ifPresent(nodeId -> this.remoteNodeId = nodeId);
```

Comment:
previous version was shorter ;)

Reply:
Yes, but this one more clear in the expression of intent.

Reply:
I disagree, but I think it's a matter of taste. I'm okay with keeping it the new way